ckanext/datawagovautheme/templates/home/snippets/search.html

```diff
@@ -15,7 +15,10 @@ <h3 class="heading">{{ _("Search data") }}</h3>
   <div class="tags">
     <h3>{{ _('Popular tags') }}</h3>
```

Comment:
We should do this for more than just tags. Organisations, Formats, and Licenses all suffer from the same issue in the sidebar on [https://uat.landgate.links.com.au/dataset](https://uat.landgate.links.com.au/dataset).

Reply:
Nevermind, I take it this is what the `facet_list.html` changes below do?

Reply:
Hey @keithmoss please note that this was accidentally committed so I reverted it back and did separate commit for #29